pilot/pkg/networking/core/v1alpha3/tracing_test.go

```diff
@@ -101,6 +105,40 @@ func TestConfigureTracing(t *testing.T) {
 	}
 }
 
+func TestSkywalkingTracingConfigure(t *testing.T) {
+	clusterName := "testcluster"
+	providerName := "foo"
+
+	clusterLookupFn = func(push *model.PushContext, service string, port int) (hostname string, cluster string, err error) {
+		return "testhost", clusterName, nil
+	}
+	defer func() {
+		clusterLookupFn = extensionproviders.LookupCluster
+	}()
+
+	testcases := []struct {
+		name     string
+		spec     *tpb.Telemetry
+		opts     buildListenerOpts
+		expected *hpb.HttpConnectionManager_Tracing
+	}{
+		{
```

Comment:
Is there any reason this couldn't be added as a testcase to the existing test?  It looks like the setup is the same, as is the actually test loop.

Reply:
As it is to be more clear to separate Zipkin and SkyWalking test.

Reply:
It is better to keep these two separated. In case of new features add in the future.

Reply:
But the other test isn't meant to be *just* a zipkin test. It is a test of generating tracing configuration. The fact that zipkin is used was just to have a placeholder fake provider. I'd much prefer to add SkyWalking to the list of fake providers and keep a single test for validating configuration than have separate repetitive tests for each provider.

If the time comes when that is no longer tenable, we can split things out then.

Reply:
Sure. I will add other providers for extension provider tests in the future (not in this PR). I will fix it just to add existing placeholder and add/separate with appending other tests such as datadog, opencensus agent... as an improvement of new Telemetry API.